portal/guides/mint-specific-nft-with-nextjs.mdx

```diff
@@ -0,0 +1,522 @@
+---
+title: Mint a Specific NFT with Next.js
+author: "Mordi Goldstein"
+authorAvatar: "/assets/portal/authors/mordi.jpg"
+authorTwitter: "https://twitter.com/mordigoldstein"
+draft: false
+tags: [nft-collection, typescript, sdk, v2]
+summary: "Allow users to select which 1-of-1 NFT to mint"
```

Comment:
```suggestion
summary: "Allow users to select which 1-of-1 NFT to mint"
```

We need a better summary here, around 150 words, explaining what the guide is doing